examples/run_nutation.py

```diff
@@ -67,79 +82,38 @@
-    nutation_data.reorder("t2", first=False)
-    for_plot = nutation_data.C
-    for_plot.ft("t2", shift=True)
-    for_plot.ft(["ph1"], unitary=True)
-    fl.next("FTed data")
-    fl.image(for_plot.C.mean("nScans"))
-else:
-    if config_dict["nScans"] > 1:
-        nutation_data.setaxis("nScans", r_[0 : config_dict["nScans"]])
-    nutation_data.rename("t", "t2")
-    fl.next("Raw - time")
-    fl.image(nutation_data.C.mean("nScans"))
-    nutation_data.reorder("t2", first=False)
-    for_plot = nutation_data.C
-    for_plot.ft("t2", shift=True)
-    fl.next("FTed data")
-    fl.image(for_plot)
-nutation_data.name(
-    config_dict["type"] + "_" + str(config_dict["echo_counter"])
-)
-nutation_data.set_prop("postproc_type", "spincore_nutation_v1")
+nutation_data.setaxis("indirect", p90_range_us * 1e-6).set_units("indirect", "s")
+# {{{ chunk and save data
+nutation_data.chunk("t", ["ph1", "t2"], [4, -1])
+nutation_data.setaxis("ph1", ph1_cyc / 4)
+if config_dict["nScans"] > 1:
+    nutation_data.setaxis("nScans", r_[0 : config_dict["nScans"]])
+nutation_data.reorder(["ph1", "nScans", "t2"])
+nutation_data.set_units("t2", "s")
+nutation_data.set_prop("postproc_type", "spincore_nutation_v4")
+nutation_data.set_prop("coherence_pathway", {"ph1": +1})
 nutation_data.set_prop("acq_params", config_dict.asdict())
-target_directory = getDATADIR(exp_type="ODNP_NMR_comp/nutation")
+nodename = config_dict["type"] + "_" + str(config_dict["echo_counter"])
+nutation_data.name(nodename)
 filename_out = filename + ".h5"
 nodename = nutation_data.name()
 if os.path.exists(f"{filename_out}"):
     print("this file already exists so we will add a node to it!")
     with h5py.File(
         os.path.normpath(os.path.join(target_directory, f"{filename_out}"))
     ) as fp:
-        if nodename in fp.keys():
-            print(
-                "this nodename already exists, so I will call it temp_nutation"
-            )
-            nutation_data.name("temp_nutation")
-            nodename = "temp_nutation"
-        nutation_data.hdf5_write(f"{filename_out}", directory=target_directory)
-else:
-    try:
-        nutation_data.hdf5_write(f"{filename_out}", directory=target_directory)
-    except:
-        print(
-            f"I had problems writing to the correct file {filename}.h5, so I'm going to try to save your file to temp_nutation.h5 in the current directory"
-        )
-        if os.path.exists("temp_nutation.h5"):
-            print("there is a temp_nutation.h5 already! -- I'm removing it")
-            os.remove("temp_nutation.h5")
-            nutation_data.hdf5_write("temp_nutation.h5")
-            print(
-                "if I got this far, that probably worked -- be sure to move/rename temp_nutation.h5 to the correct name!!"
-            )
+        while nodename in fp.keys():
+            nodename = config_dict["type"] + "_" + str(config_dict["echo_counter"])
+            nutation_data.name(nodename)
+            config_dict["echo_counter"] += 1
```

Comment:
I realized this is a little goofy.  It should be
```
while nodename in fp.keys():
    config_dict["echo_counter"] += 1
    nodename = config_dict["type"] + "_" + str(config_dict["echo_counter"])
nutation_data.name(nodename)
```

Reply:
rather than changing this 3x, it's most likely worth it to again make a function that does all this.

Reply:
(all the way down to printing the file info, and even running proc_raw, if you want to do that).

Reply:
again -- sorry that this is something I didn't catch earlier --> it's a general rule that when you're doing the exact same thing in 3 different places, you should make a function to do that thing, so we should do that here.

Reply:
we diecided to make this into a separate PR